test/service/test_request_validation.py

```diff
@@ -153,6 +153,20 @@ def test_valid_sort_facet_but_bad_filter_facet_of_file(self):
         self.assertEqual(400, response.status_code, response.json())
         self.assertEqual(self.filter_facet_message, response.json())
 
+    def test_single_entity_error_responses(self):
+        self._index_canned_bundle(("f79257a7-dfc6-46d6-ae00-ba4b25313c10", "2018-09-14T133314.453337Z"))
+        entities = ['specimens', 'files', 'projects']
+        expected_values = [('2b7959bb-acd1-4aa3-9557-345f9b3c6327', 404),
+                           ('-0c5ac7c0-817e-40d4-b1b1-34c3d5cfecdb-', 400),
+                           ('FOO', 400)]
+
+        for uuid, expected_error_code in expected_values:
+            for entity_name in entities:
```

Comment:
```suggestion
            for entity_type in entity_types:
```

The degree of variation and inconsistency in your variable naming is impressive.